crates/swc/src/config/mod.rs

```diff
@@ -971,6 +971,13 @@ pub struct JscExperimental {
     /// If true, keeps import assertions in the output.
     #[serde(default)]
     pub keep_import_assertions: bool,
+    /// Location where swc may stores its intermediate cache.
+    /// Currently this is only being used for wasm plugin's bytecache.
+    /// Path should be absolute directory, which will be created if not exist.
+    /// This configuration behavior can change anytime under experimental flag
+    /// and will not be considered as breaking changes.
+    #[serde(default)]
+    pub cache_root: Option<String>,
```

Comment:
I don't have clear pictures around how we'll expose configuration around cache yet, so put it under experimental flag. Easiest way is as current does accepting a string to the path - but maybe we want to some level of configuration (i.e what should we do if we want to opt in mem cache? or control over each cache targets instead of root?) instead.

Reply:
Yeah, I'm also not sure about this at the moment,

Reply:
I think it should be `plugin_cache_name`. (Or something includes `plugin`)

Reply:
I was thinking more broad to not to include `plugin` explicitly:

```
{
..
cache_root: //some_path
}
```

then there can be

```
- cache_root
   - plugins
   - some other for future, i.e bundle, etcs..
```

even if we have single cache forever for plugin only, I think it's better to have root level config instead.